decidim-comments/app/frontend/comments/comment.component.tsx

```diff
@@ -76,6 +76,11 @@ class Comment extends React.Component<CommentProps, CommentState> {
                   <img src={author.avatarUrl} alt="author-avatar" />
                 </a>
                 <a className="author__name">{author.name}</a>
+                { author.isVerified &&
+                  <span>&nbsp;
+                    <Icon name="icon-check" iconExtraClassName="icon--small" />
```

Comment:
We should add a `title` or something so if I hover over it, it tells that this means the users has been authorized.